File: UI.MVC/Models/Dto/ProjectDto.cs
using System.ComponentModel.DataAnnotations;

namespace UI.MVC.Models.Dto;

public class ProjectDto
{
    [Required(ErrorMessage = "Naam is verplicht.")]
    public string Name { get; set; }

    [Required(ErrorMessage = "Beschrijving is verplicht.")]
    public string Description { get; set; }

    public long ProjectId { get; set; }

    public bool Circulair { get; set; }
    [Required(ErrorMessage = "Selecteer of het actief moet zijn of niet.")]
    public bool isActive { get; set; }

    public string BackgroundColor { get; set; }

    public string Font { get; set; }
}

File: UI.MVC/ClientApp/src/ts/Configs/ProjectConfig/ProjectDto.ts
export interface ProjectDto {
    Circulair: boolean;
    projectId: number;
    name: string;
    description: string;
    isActive: boolean;
    backgroundColor: string;
    font: string;
}

export interface flowdto
{
    flowId : number;
    title : string;
}
export interface projectPost {
    projectId: number;
    Circulair: boolean;
}